app/controllers/avo/application_controller.rb

```diff
@@ -25,6 +25,7 @@ class ApplicationController < ::ActionController::Base
     before_action :set_view
     before_action :set_sidebar_open
     before_action :set_stylesheet_assets_path
+    before_action :set_paper_trail_whodunnit, if: -> { defined? PaperTrail }
```

Comment:
Can't we add this ourselves in the audit logging gem?

Reply:
Makes sense, done